packages/neon/neon_notes/lib/sort/notes.dart

```diff
@@ -4,8 +4,14 @@ final notesSortBox = SortBox<NotesSortProperty, NotesNote>(
   {
     NotesSortProperty.alphabetical: (final note) => note.title.toLowerCase(),
     NotesSortProperty.lastModified: (final note) => note.modified,
+    NotesSortProperty.favorite: (final note) => note.favorite ? 0 : 1,
   },
   {
-    NotesSortProperty.alphabetical: Box(NotesSortProperty.lastModified, SortBoxOrder.descending),
+    NotesSortProperty.alphabetical: {
+      Box(NotesSortProperty.lastModified, SortBoxOrder.descending),
+    },
+    NotesSortProperty.lastModified: {
+      Box(NotesSortProperty.lastModified, SortBoxOrder.descending),
+    },
```

Comment:
This doesn't make sense as it will sort by last modified on the first and second level. I assume you meant to put something else there? I'd go with alphabetical ascending in this case.